$NetBSD: patch-CMakeLists.txt,v 1.1 2014/04/30 23:42:52 wiz Exp $

Always use embedded libav, since there is no pkgsrc package for
libav and the ffmpeg package is misdetected as libav.
https://github.com/andrewrk/libgroove/issues/58

--- CMakeLists.txt.orig	2014-04-21 07:09:21.000000000 +0000
+++ CMakeLists.txt
@@ -121,7 +121,7 @@ endif(DISABLE_PLAYER)
 
 # check for libav
 find_package(LibAV 10.0)
-if(LIBAV_AVCODEC_FOUND AND LIBAV_AVFILTER_FOUND AND LIBAV_AVFORMAT_FOUND AND LIBAV_AVUTIL_FOUND)
+if(LIBAV_AVCODEC_FOUND AND LIBAV_AVFILTER_FOUND AND LIBAV_AVFORMAT_FOUND AND LIBAV_AVUTIL_FOUND AND FALSE)
   set(LIBAV_IS_BUNDLED FALSE)
   set(STATUS_DEP_LIBAV "using system libraries")
   set(STATUS_BZIP2 "not needed")
